homeassistant/components/glances/sensor.py

```diff
@@ -24,7 +24,20 @@
     dev = []
     for sensor_type in SENSOR_TYPES:
         dev.append(
-            GlancesSensor(glances_data, name, SENSOR_TYPES[sensor_type][0], sensor_type)
+            GlancesSensor(
+                glances_data,
+                name,
+                SENSOR_TYPES[sensor_type][0],
+                sensor_type,
+                SENSOR_TYPES[sensor_type],
+            )
+        )
+
+    for sensor in glances_data.api.data["sensors"]:
+        dev.append(
+            GlancesSensor(
+                glances_data, name, sensor["label"], CPU_TEMP, TEMP_SENSORS[CPU_TEMP]
```

Comment:
We should adjust the naming as we are no longer only working with CPU temperatures but chipsets and other peripheral hardware.

Reply:
Ok, I can change this to `sensor_temp`. Or do you recommend another naming?

Reply:
I'm fine with `sensor_temp`.